sdk/src/main/java/io/horizen/sc2sc/Sc2ScConfigurator.scala

```diff
@@ -0,0 +1,4 @@
+package io.horizen.sc2sc
+
+case class Sc2ScConfigurator(canSendMessages: Boolean, canReceiveMessages: Boolean)
```

Comment:
see my comments in AbstractSidechainApp/SidechainApp classes.
We must care about backward compatibility.
`canSendMessages` and `canReceiveMessages` in general can be activated after genesis block and even separately, optionally:
1) A newly created sidechain may activate both sender and receiver at the genesis phase
2) EON will do it after genesis
3) other chain may decide to skip this feature.
It is a for sure a new mechanism for our SDK and ForkManager.